### ISSUE TITLE
TypeError: Cannot read property 'current' of undefined

### ISSUE THREAD

**Issue by arelaxend** [ugogo/react-input-verification-code#1]
Hi,

Nice! Please continue

From time to time, it throws "TypeError: Cannot read property 'current' of undefined":

```
TypeError: Cannot read property 'current' of undefined
getItem
src/index.tsx:33
  30 | 
  31 | const isCodeRegex = new RegExp(`^[0-9]{${length}}$`);
  32 | 
> 33 | const getItem = (index: number) => itemsRef[index].current!;
     | ^  34 | const focusItem = (index: number): void => getItem(index).focus();
  35 | const blurItem = (index: number): void => getItem(index).blur();
```

**How to reproduce ?**
Wait some time on the codesandox example and click on the different boxes

Best, A

**Comment by ugogo** [comment]
Hi @arelaxend,

Thanks for reporting!
I'll have a look

**Comment by ugogo** [comment]
**Steps to reproduce**
- Focus an item
- Switch browser tab
- Come back to the tab
- Click anywhere

**Comment by ugogo** [comment]
This should be fixed in [0.4.4](https://github.com/ugogo/react-input-verification-code/releases/tag/0.4.4)
Closing this issue, feel free to re-open it if needed
Thanks for reporting this 🙏
